bucket/butane.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.11.0",
+    "description": "Butane translates human-readable Butane configs into machine-readable Ignition configs. Formerly known as Fedora CoreOS Config Transpiler (FCCT).",
```

Comment:
```suggestion
    "description": "Fedora CoreOS Config Transpiler, FCCT. Translates Butane configs into Ignition configs.",
```

Reply:
The project isn't called `Fedora CoreOS Config Transpiler (FCCT)` and that shouldn't be the main takeaway from the description. In case of Homebrew, the [formula](https://formulae.brew.sh/formula/butane) received a separately tracked name change but other than that doesn't refer to the outdated name at all and the [MacPorts](https://ports.macports.org/port/butane/summary) version has no reference to it because it was introduced afterwards.

Reply:
@Ash258 what are we going to do about this? If you absolutely insist on putting the outdated description in … I guess I'll oblige despite my reservations.